xapi-tracing.opam.template

```diff
@@ -1,26 +1 @@
-opam-version: "2.0"
-maintainer: "xen-api@lists.xen.org"
-authors: "xen-api@lists.xen.org"
-homepage: "https://xapi-project.github.io/"
-bug-reports: "https://github.com/xapi-project/xen-api.git"
-dev-repo: "git+https://github.com/xapi-project/xen-api.git"
-build: [[ "dune" "build" "-p" name "-j" jobs ]]
-
 available: [ os = "linux" ]
```

Comment:
so the template still exists even if we are generating the opam from dune-project? Will dune look at this template when generating?